splunk-apm/README.md

```diff
@@ -0,0 +1,86 @@
+# Splunk distribution of OpenTelemetry Lambda
+
+Splunk OTEL lambda is a downstream distribution of opentelemetry-lambda. Supported OpenTelemetry Lambda layers are preconfigured for easy use with Splunk environment (ingest, SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](https://github.com/signalfx/lambda-layer-versions/).
+
+## Get started 
+
+Currently, following components are supported:
+- [Java wrapper](#java-wrapper) 
+
+### Configuration
+
+All wrapper layers come preconfigured for quick start with Splunk.
+
+1. Context propagation
+    By default, W3C specified Trace Context and Baggage (`tracecontext,baggage`) context propagation are used. 
+    
+    If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your Lambda function code. For more information about available context propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v1.1.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk) for the OpenTelemetry Java.
+
+2. Traces export
+    By default, the OpenTelemetry Protocol (`otlp`) exporter is used.
+    If you want to use this exporter, set these environment variables in your Lambda function code:
+    ```
+    OTEL_EXPORTER_OTLP_ENDPOINT=<Endpoint URL>
+    SPLUNK_ACCESS_TOKEN="orgAccessToken"
+    ```
+    Splunk provides also token-authenticated `jaeger-thrift-splunk` exporter for customers that need to use that specific protocol. In order to use it, please set (example endpoint value for SmartAgent):
+    ```
+    OTEL_TRACES_EXPORTER=jaeger-thrift-splunk
+    OTEL_EXPORTER_JAEGER_ENDPOINT=<Endpoint URL>
+    SPLUNK_ACCESS_TOKEN="orgAccessToken"
+    ``` 
+    By default, `OTEL_EXPORTER_JAEGER_ENDPOINT` is set to `http://127.0.0.1:9080/v1/trace` (local SmartAgent endpoint)
+   
+    Default span flush wait timeout is 1 second. It is controlled with a following property (value in milliseconds):
+    ```
+    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
+    ```
```

Comment:
```suggestion
2. Traces export
    This distribution uses the OpenTelemetry Protocol (`otlp`) exporter by default. 
    
    To use this exporter, set these environment variables in your Lambda function code:
    
    ```
    OTEL_EXPORTER_OTLP_ENDPOINT=<EndpointURL>
    SPLUNK_ACCESS_TOKEN="<orgAccessToken>"
    ```
    Splunk also provides a token-authenticated `jaeger-thrift-splunk` exporter for customers that require it. The following example shows how to set the environment variables pointing to the Smart Agent endpoint:

    ```
    OTEL_TRACES_EXPORTER=jaeger-thrift-splunk
    OTEL_EXPORTER_JAEGER_ENDPOINT=<Endpoint URL>
    SPLUNK_ACCESS_TOKEN="orgAccessToken"
    ``` 

  The default value of `OTEL_EXPORTER_JAEGER_ENDPOINT` is `http://127.0.0.1:9080/v1/trace`, which corresponds to the local Smart Agent endpoint.
   
    Default flush wait timeout for spans is 1 second. It can be controlled through the following property (value in milliseconds):

    ```
    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
    ```
```

Reply:
Shouldn't we be moving away from Smart Agent instructions? Shouldn't we provide an Ingest endpoint example instead (the only valid use case for sending data using the Jaeger exporter, at least until support for OTLP http is gold).